### DISCUSSION TITLE
German translation

### DISCUSSION THREAD

**Discussion by mhotzel** [israel-dryer/ttkbootstrap#425] [Ideas]
Hey,

I would like to share a german translation for /src/ttkbootstrap/localization/msgs.py:
`
MESSAGES.append(
    LocaleMsgs(
        # German
        "de",
        ("OK", "OK"),
        ("Ok", "Ok"),
        ("Retry", "Wiederholen"),
        ("Delete", "Löschen"),
        ("Next", "Nächster"),
        ("Prev", "Vorher."),
        ("Yes", "Ja"),
        ("No", "Nein"),
        ("Open", "Öffnen"),
        ("Close", "Schließen"),
        ("Add", "Hinzufügen"),
        ("Remove", "Entfernen"),
        ("Submit", "Absenden"),
        ("Family", "Familie"),
        ("Weight", "Gewicht"),
        ("Slant", "schräg"),
        ("Effects", "Effekte"),
        ("Preview", "Vorschau"),
        ("Size", "Größe"),
        ("Should be a of data type", "Sollte ein Datentyp sein"),
        ("Invalid data type", "Ungültiger Datentyp"),
        ("Number cannot be greater than", "Zahl darf nicht größer sein als"),
        ("Out of range", "Außerhalb des Bereiches"),
        ("Previous", "Vorheriger"),
        ("Font Selector", "Schriftartauswahl"),
        ("normal", "normal"),
        ("bold", "fett"),
        ("roman", "römisch"),
        ("italic", "kursiv"),
        ("underline", "unterstrichen"),
        ("overstrike", "überstrichen"),
        ("Color Chooser", "Farbwähler"),
        ("Advanced", "Erweitert"),
        ("Themed", "Themen"),
        ("Standard", "Standard"),
        ("Current", "Aktuell"),
        ("New", "Neu"),
        ("Hue", "Frbton"),
        ("Sat", "Sättigung"),
        ("Lum", "Leuchtkraft"),
        ("Hex", "Hex"),
        ("Red", "Rot"),
        ("Green", "Grün"),
        ("Blue", "Blau"),
        ("color dropper", "Farbpipette"),
        ("Cancel", "Abbruch"),
        ("Search", "Suchen"),
        ("Page", "Seite"),
        ("of", "von"),
        ("⎌", "↺"),
        ("Reset table", "Tabelle rücksetzen"),
        ("Columns", "Spalten"),
        ("Move", "Verschieben"),
        ("Align", "Ausrichten"),
        ("Hide column", "Spalte ausblenden"),
        ("Delete column", "Spalte löschen"),
        ("Show All", "Alles anzeigen"),
        ("Move to left", "Nach links bewegen"),
        ("Move to right", "Nach rechts bewegen"),
        ("Move to first", "Zum ersten gehen"),
        ("Move to last", "Zum letzten gehen"),
        ("Align left", "Links ausrichten"),
        ("Align center", "Mittig ausrichten"),
        ("Align right", "Rechts ausrichten"),
        ("Sort", "Sortieren"),
        ("Filter", "Filtern"),
        ("Export", "Exportieren"),
        ("Delete selected rows", "Ausgewählte Zeilen löschen"),
        ("Sort Ascending", "Aufsteigend sortieren"),
        ("Sort Descending", "Absteigend sortieren"),
        ("Clear filters", "Filter löschen"),
        ("Filter by cell's value", "Nach Zellenwert filtern"),
        ("Hide select rows", "Ausgewählte Zeilen ausblenden"),
        ("Show only select rows", "Nur selektierte Zeilen anzeigen"),
        ("Export all records", "Alle Datensätze exportieren"),
        ("Export current page", "Aktuelle Seite exportieren"),
        ("Export current selection", "Auswahl exportieren"),
        ("Export records in filter", "Gefilterte Einträge exportieren"),
        ("Move up", "Aufwärts bewegen"),
        ("Move down", "Abwärts bewegen"),
        ("Move to top", "Ganz nach oben"),
        ("Move to bottom", "Ganz nach unten"),
        ("Mo", "Mo"),
        ("Tu", "Di"),
        ("We", "Mi"),
        ("Th", "Do"),
        ("Fr", "Fr"),
        ("Sa", "Sa"),
        ("Su", "So"),
    )
)
`

**Comment by vmakrov1720** [comment]
Hello mhotzel, can you show me a simple sample.py to use this translation. Thanks.

**Comment by TheDemonGuard** [comment]
@mhotzel 
The German translation as already been added by PR #437 

> Hey,
> 
> I would like to share a german translation for /src/ttkbootstrap/localization/msgs.py: `MESSAGES.append( LocaleMsgs( # German "de", ("OK", "OK"), ("Ok", "Ok"), ("Retry", "Wiederholen"), ("Delete", "Löschen"), ("Next", "Nächster"), ("Prev", "Vorher."), ("Yes", "Ja"), ("No", "Nein"), ("Open", "Öffnen"), ("Close", "Schließen"), ("Add", "Hinzufügen"), ("Remove", "Entfernen"), ("Submit", "Absenden"), ("Family", "Familie"), ("Weight", "Gewicht"), ("Slant", "schräg"), ("Effects", "Effekte"), ("Preview", "Vorschau"), ("Size", "Größe"), ("Should be a of data type", "Sollte ein Datentyp sein"), ("Invalid data type", "Ungültiger Datentyp"), ("Number cannot be greater than", "Zahl darf nicht größer sein als"), ("Out of range", "Außerhalb des Bereiches"), ("Previous", "Vorheriger"), ("Font Selector", "Schriftartauswahl"), ("normal", "normal"), ("bold", "fett"), ("roman", "römisch"), ("italic", "kursiv"), ("underline", "unterstrichen"), ("overstrike", "überstrichen"), ("Color Chooser", "Farbwähler"), ("Advanced", "Erweitert"), ("Themed", "Themen"), ("Standard", "Standard"), ("Current", "Aktuell"), ("New", "Neu"), ("Hue", "Frbton"), ("Sat", "Sättigung"), ("Lum", "Leuchtkraft"), ("Hex", "Hex"), ("Red", "Rot"), ("Green", "Grün"), ("Blue", "Blau"), ("color dropper", "Farbpipette"), ("Cancel", "Abbruch"), ("Search", "Suchen"), ("Page", "Seite"), ("of", "von"), ("⎌", "↺"), ("Reset table", "Tabelle rücksetzen"), ("Columns", "Spalten"), ("Move", "Verschieben"), ("Align", "Ausrichten"), ("Hide column", "Spalte ausblenden"), ("Delete column", "Spalte löschen"), ("Show All", "Alles anzeigen"), ("Move to left", "Nach links bewegen"), ("Move to right", "Nach rechts bewegen"), ("Move to first", "Zum ersten gehen"), ("Move to last", "Zum letzten gehen"), ("Align left", "Links ausrichten"), ("Align center", "Mittig ausrichten"), ("Align right", "Rechts ausrichten"), ("Sort", "Sortieren"), ("Filter", "Filtern"), ("Export", "Exportieren"), ("Delete selected rows", "Ausgewählte Zeilen löschen"), ("Sort Ascending", "Aufsteigend sortieren"), ("Sort Descending", "Absteigend sortieren"), ("Clear filters", "Filter löschen"), ("Filter by cell's value", "Nach Zellenwert filtern"), ("Hide select rows", "Ausgewählte Zeilen ausblenden"), ("Show only select rows", "Nur selektierte Zeilen anzeigen"), ("Export all records", "Alle Datensätze exportieren"), ("Export current page", "Aktuelle Seite exportieren"), ("Export current selection", "Auswahl exportieren"), ("Export records in filter", "Gefilterte Einträge exportieren"), ("Move up", "Aufwärts bewegen"), ("Move down", "Abwärts bewegen"), ("Move to top", "Ganz nach oben"), ("Move to bottom", "Ganz nach unten"), ("Mo", "Mo"), ("Tu", "Di"), ("We", "Mi"), ("Th", "Do"), ("Fr", "Fr"), ("Sa", "Sa"), ("Su", "So"), ) )`
